README.md

```diff
@@ -171,42 +199,41 @@ The recommended RSpec configuration is as follows:
 
 ```ruby
 # spec/support/pub_sub.rb
- 
+
 require 'pub_sub/testing'
 
 RSpec.configure do |config|
-  config.include Wisper::RSpec::BroadcastMatcher
-  config.include PubSub::Testing::SubscriptionsHelper
+  config.include PubSub::Testing::RailsEventStore
   config.include PubSub::Testing::EventDataHelper
 
-  config.before(:suite) do
-    PubSub::Testing::SubscriptionsHelper.clear_wisper_subscriptions!
-  end
-
-  config.around(:each, subscribers: true) do |example|
-    domain_name = example.metadata[:described_class].to_s.underscore
-    PubSub.subscriptions.register(domain_name)
+  config.around(:each, in_memory_res_client: true) do |example|
+    current_event_store = Rails.configuration.event_store
+    Rails.configuration.event_store = RubyEventStore::Client.new(
+      repository: RubyEventStore::InMemoryRepository.new
+    )
     example.run
-    clear_wisper_subscriptions!
+    Rails.configuration.event_store = current_event_store
   end
 end
 ```
 
+This will gives you an opportunity to use `in_memory_res_client` which will not create object in the database and do not call all dependent logic.
```

Comment:
```suggestion
This will allow you to use `in_memory_res_client` which will not create object (event) in the database and do not call all dependent logic (handlers).
```